azure-pipelines.yml

```
@@ -380,6 +374,89 @@ jobs:
       docker run $IMAGE_NAME /bin/bash -c "time az self-test && time az --version && sleep 5"
     displayName: 'Bash Script'
 
+- job: BuildExtraDockerImage
+  displayName: Build Extra Docker Image
+  dependsOn: BuildRpmPackageMariner
```

This job should be renamed:

```suggestion
  dependsOn: BuildRpmPackageAzureLinux
```

---

`Mariner` is still used in pipeline. Let's wait for Azure Linux team change the url of docker image.